core/domain/platform_feature_services.py

```diff
@@ -270,9 +270,13 @@ def _evaluate_feature_flag_values_for_context(
 
     result_dict = {}
     for feature_name in feature_names_set:
-        param = registry.Registry.get_platform_parameter(
+        feature = registry.Registry.get_platform_parameter(
             feature_name)
-        feature_name_value = param.evaluate(context)
+        assert feature.feature_stage is not None
```

Comment:
Why is this assertion needed?

Reply:
Not relevant anymore, removed after new changes.